[debug] [2018-11-09T17:37:21.574Z] ----------------------------------------------------------------------
[debug] [2018-11-09T17:37:21.576Z] Command:       /usr/local/bin/node /usr/local/bin/firebase functions:shell
[debug] [2018-11-09T17:37:21.577Z] CLI Version:   4.2.1
[debug] [2018-11-09T17:37:21.577Z] Platform:      darwin
[debug] [2018-11-09T17:37:21.577Z] Node Version:  v8.11.3
[debug] [2018-11-09T17:37:21.577Z] Time:          Fri Nov 09 2018 11:37:21 GMT-0600 (CST)
[debug] [2018-11-09T17:37:21.577Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-11-09T17:37:21.591Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-11-09T17:37:21.591Z] > authorizing via signed-in user
[debug] [2018-11-09T17:37:21.593Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/dj-prudo  
 
[debug] [2018-11-09T17:37:21.820Z] <<< HTTP RESPONSE 200
[debug] [2018-11-09T17:37:22.609Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:113:21)
    at Command.module.exports [as _action] (/usr/local/lib/node_modules/firebase-tools/lib/functionsShellCommandAction.js:18:6)
    at /usr/local/lib/node_modules/firebase-tools/lib/command.js:202:29
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2018-11-09T17:40:31.985Z] ----------------------------------------------------------------------
[debug] [2018-11-09T17:40:31.989Z] Command:       /usr/local/bin/node /usr/local/bin/firebase serve --only functions
[debug] [2018-11-09T17:40:31.989Z] CLI Version:   4.2.1
[debug] [2018-11-09T17:40:31.990Z] Platform:      darwin
[debug] [2018-11-09T17:40:31.990Z] Node Version:  v8.11.3
[debug] [2018-11-09T17:40:31.991Z] Time:          Fri Nov 09 2018 11:40:31 GMT-0600 (CST)
[debug] [2018-11-09T17:40:31.991Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-11-09T17:40:32.005Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-11-09T17:40:32.006Z] > authorizing via signed-in user
[debug] [2018-11-09T17:40:32.008Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/dj-prudo  
 
[debug] [2018-11-09T17:40:32.313Z] <<< HTTP RESPONSE 200
[debug] [2018-11-09T17:40:32.314Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/dj-prudo  
 
[debug] [2018-11-09T17:40:32.462Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/Users/sumin/Documents/EECS497/blue-project2'...
[info] 
[debug] [2018-11-09T17:40:33.314Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:113:21)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:9:29)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:17:21
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:7)
    at Command._action (/usr/local/lib/node_modules/firebase-tools/commands/serve.js:48:12)
[error] 
[error] Error: An unexpected error has occurred.
